### PR TITLE
Catty 71 switch all existing sound bricks to new audio engine

### PR DESCRIPTION
**Pull request by Wullschi**

### CODE REVIEW

**Comment by m-herold** [comment]
PO review: to sum up the instructions for the following Bricks changed:

- ChangeVolumeByNBrick
- PlaySoundBrick
- SetVolumeToBrick
- StopAllSoundsBrick

File: src/CattyTests/Bricks/SetYBrickTests.swift [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+        self.scene.addChild(spriteNode)
+        spriteNode.catrobatPosition = CGPoint(x: 0, y: 0)
+
+        script = WhenScript()
+        script.object = object
+
+        brick = SetYBrick()
+        brick.script = script
+
```
Comment (GatCode) [inline src/CattyTests/Bricks/SetYBrickTests.swift]: space

File: src/CattyTests/PlayerEngine/AudioEngine/Helpers/AudioEngineAbstractTest.swift [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    var audioEngine: AudioEngineFingerprintingStub!
+    var recorder: AKNodeRecorder!
+
+    override func setUp() {
+        super.setUp()
+        do {
+            tape = try AKAudioFile()
+            audioEngine = AudioEngineFingerprintingStub(audioPlayerFactory: FingerprintingAudioPlayerFactory())
+            recorder = audioEngine.addNodeRecorderAtEngineOut(tape: tape)
+
```
Comment (GatCode) [inline src/CattyTests/PlayerEngine/AudioEngine/Helpers/AudioEngineAbstractTest.swift]: space

File: src/CattyTests/PlayerEngine/AudioEngine/Mocks/FingerprintingAudioPlayerFactory.swift [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+        do {
+            let file = try AKAudioFile(forReading: audioFileURL)
+            audioPlayer = AudioPlayer(soundFile: file, addCompletionHandler: true)
+        } catch {
+            print("Could not load audio file with url \(audioFileURL.absoluteString)")
+        }
+
+        return audioPlayer
+    }
+
```
Comment (GatCode) [inline src/CattyTests/PlayerEngine/AudioEngine/Mocks/FingerprintingAudioPlayerFactory.swift]: space

**Pull request merged**